The RV application plugin test fails

## Bug Report
### Incorrect Functionality and General Questions

The RV application plugin test fails when it is run.  Output:
```
======================================================================
FAIL: test_read_otio_file (test_rv_reader.RVSessionAdapterReadTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/nelsonr/git/public/OpenTimelineIO/contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py", line 219, in test_read_otio_file
    self.assertEqual(clip2, 'clip2.mov')
AssertionError: 'clip3.mov' != 'clip2.mov'

----------------------------------------------------------------------
Ran 3 tests in 14.975s
```

## To Reproduce

1. Operating System: Mac OS 11.6
2. Python version: System (Python 2.7.16)
3. Example snippet that demonstrates the issue - if it's a build issue: 
4. OpenTimelineIO release version or commit hash: 1ba6d78
5. Compiler information: Apple LLVM version 10.0.0 (clang-1000.11.45.2)

## Expected Behavior
The tests pass

--- a/contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py
+++ b/contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py
@@ -84,7 +84,7 @@ for clipnum in range(1, 4):
             ),
             source_range=otio.opentime.TimeRange(
                 otio.opentime.RationalTime(11, 24),
-                otio.opentime.RationalTime(30, 24)
+                otio.opentime.RationalTime(3, 24)
             )
         )
     )
@@ -205,10 +205,10 @@ class RVSessionAdapterReadTest(unittest.TestCase):
             clip1 = rv_media_name_at_frame(rvc, 1)
             self.assertEqual(clip1, 'clip1.mov')
 
-            clip2 = rv_media_name_at_frame(rvc, 20)
+            clip2 = rv_media_name_at_frame(rvc, 4)
             self.assertEqual(clip2, 'clip2.mov')
 
-            clip3 = rv_media_name_at_frame(rvc, 40)
+            clip3 = rv_media_name_at_frame(rvc, 7)
             self.assertEqual(clip3, 'clip3.mov')
 
             rvc.disconnect()

